docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,428 @@
+        id3-- host with high CPU -->id4(Kafka sink);
+
+
+1. In the Aiven CLI, run the following command to connect to the ``demo-postgresql`` service:
+   
+   ::
+	  
+      avn service cli demo-postgresql --project PROJECT_NAME
+   
+#. Enter the following commands to set up the PostgreSQL table containing the threshold values:
+   
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 52-53
+      :language: sql
+
+#. Enter the following command to check that the threshold values are created:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 54
+      :language: sql
+
+   The output shows you the content of the table:
+
+   ::
+
+      hostname | allowed_top
+      ---------+------------
+      doc      |     20
+      grumpy   |     30
+      sleepy   |     40
+      bashful  |     60
+      happy    |     70
+      sneezy   |     80
+      dopey    |     90
+
+#. Run the following command to create a PostgreSQL table named ``SOURCE_THRESHOLDS``:
+
+   ::
+
+      avn service flink table create demo-flink POSTGRESQL_INTEGRATION_ID  \
+          --table-name SOURCE_THRESHOLDS                                   \
+          --jdbc-table cpu_thresholds                                      \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``POSTGRESQL_INTEGRATION_ID`` with the ID for your ``demo-postgresql`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 57-59
+      :language: sql
+
+#. Run the following command to create a Kafka table named ``CPU_OUT_FILTER_PG``:
+
+   ::
+
+      avn service flink table create demo-flink KAFKA_INTEGRATION_ID  \
+          --table-name CPU_OUT_FILTER_PG                              \
+          --kafka-topic cpu_load_stats_real_filter_pg                 \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``KAFKA_INTEGRATION_ID`` with the ID for your ``demo-kafka`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 62-66
+      :language: sql
+
+#. Run the following command to list the tables for the ``demo-flink`` service and get the IDs for the ``CPU_IN``, ``CPU_OUT_FILTER_PG``, and ``SOURCE_THRESHOLDS`` tables:
+
+   ::
+
+      avn service flink table list demo-flink
+
+#. Run the following command to create a data pipeline job named ``simple_filter_pg``:
+
+   ::
+
+      avn service flink job create demo-flink simple_filter_pg            \
+          --table-ids CPU_IN_ID CPU_OUT_FILTER_PG_ID SOURCE_THRESHOLDS_ID \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN``, ``CPU_OUT_FILTER_PG``, and ``SOURCE_THRESHOLDS`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 69-77
+      :language: sql
+
```

Comment:
At this point, I seem to have manually replaced a lot of IDs and SQL into commands, what do you think about using more subheadings with each section starting with a short description of what we're adding now and how it fits the big picture?

Reply:
I suppose we could, but the full set of table IDs required for each job are only available after completing the steps. I would think that it might be a bit confusing to see IDs for tables that don't yet exist at the start of the section, so it might be better to list them separately in the relevant steps.